### PR TITLE
Adding claim rewards for holders tool

### PR DESCRIPTION
**Pull request by gchatz22**

### CODE REVIEW

File: handlers/useHandleExecuteTransactions.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import type { Dispatch, SetStateAction } from 'react'
+import { useMutation } from 'react-query'
+
+const BATCH_SIZE = 50
+
+export const chunkArray = (arr: any[], size: number): any[][] =>
+  arr.length > size
+    ? [arr.slice(0, size), ...chunkArray(arr.slice(size), size)]
+    : [arr]
+
```
Comment (jpbogle) [inline handlers/useHandleExecuteTransactions.ts:16]: this one is in @cardinal/common

File: handlers/useHandleTransferFunds.ts [hunk 2]
```diff
@@ -60,6 +60,13 @@ export const useHandleTransferFunds = () => {
         true
       )
 
+      if (
+        rewardDistributor.data.parsed.rewardMint.toString() ===
+        'So11111111111111111111111111111111111111112'
+      ) {
+        await withWrapSol(transaction, connection, wallet, transferAmount)
+      }
```
Comment (jpbogle) [inline handlers/useHandleTransferFunds.ts:68]: 👍

File: generators/useGenerateClaimRewardsForHoldersTxs.ts [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        ).filter((entry) => !entry.parsed.cooldownStartSeconds)
+      }
+
+      console.log(
+        `Estimated SOL needed to claim rewards for ${
+          [...stakeEntriesV1, ...stakeEntriesV2].length
+        } staked tokens:`,
+        0.002 * [...stakeEntriesV1, ...stakeEntriesV2].length,
+        'SOL'
+      )
```
Comment (jpbogle) [inline generators/useGenerateClaimRewardsForHoldersTxs.ts:60]: shud we put this in the UI somewhere

Comment (gchatz22) [reply generators/useGenerateClaimRewardsForHoldersTxs.ts:60]: yah will add as a notification

**Review by jpbogle — APPROVED** [review]

**Pull request merged**